attic/training/abfe.py

```diff
@@ -29,7 +29,7 @@
 from rdkit.Chem import AllChem
 from rdkit.Chem import rdFMCS
 
-from fe import dataset
+from timemachine.training import dataset
```

Comment:
this change looks a little suspicious, just wanted to verify it's correct? (moving from `fe`, would've expected from `training`)

Reply:
This was broken in the original script, fixed it and updated it in one step. This scripts does not currently run for earlier reasons (broken reference on L16 for example), but figured I'd fix any references I touched during the move.